### PR TITLE
COMMERCE-9743

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-9743

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase [hunk 1]
```diff
@@ -165,6 +165,110 @@ definition {
...
+			CommerceProducts.openProductsAdmin();
+
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "Simple_Product");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceSKUs.generateAllSkuCombinations();
+		}
+
+		task ("And the admin navigates to the Subscription tab in a product Simple_Product") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase:216]: And the admin navigates to the Subscription tab of the Simple_Product

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase [hunk 2]
```diff
@@ -165,6 +165,110 @@ definition {
...
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("When the buyer navigates to the product details page") {
+			CommerceNavigator.gotoProductDetails(productName = "Simple_Product");
+		}
+
+		task ("Then the Paymente Subscription is shown correctly") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSubscriptions.testcase:259]: Then the Payment Subscription is shown correctly

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look at my comments. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by jaydawu** [comment]
Fix test and resent on https://github.com/liferay-commerce/liferay-portal/pull/4450

**Comment by jaydawu** [comment]
ci:close

**Pull request closed without merging**